tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -146,14 +147,14 @@ public function testEventManagerPassedToPlatform() : void
     {
         $eventManager = new EventManager();
 
-        /** @var AbstractPlatform|MockObject $driver */
         $platform = $this->createMock(AbstractPlatform::class);
+        assert($platform instanceof AbstractPlatform || $platform instanceof MockObject);
```

Comment:
This sniff will be disabled in tests. There's no point in making assertions against what PHPUnit generates.